File: Java/LogLevels.java
public class LogLevels {
    
    public static String message(String logLine) {
        var a = logLine.lastIndexOf(":")+1;
        return logLine.substring(a).trim();
    }
    public static String logLevel(String logLine) {
        var b = logLine.lastIndexOf("[")+1;
        var c = logLine.lastIndexOf("]");
        return logLine.substring(b,c).trim().toLowerCase();
    }
    public static String reformat(String logLine) {
        var a = logLine.lastIndexOf(":")+1;
        var message = logLine.substring(a).trim();
        var b = logLine.lastIndexOf("[")+1;
        var c = logLine.lastIndexOf("]");
        var level = logLine.substring(b,c).trim().toLowerCase();
        return message+" ("+level+")";
    }
}

File: C-Sharp/Log Levels.cs
using System;
static class LogLine
{
    public static string Message(string logLine)
    {
        if(logLine.StartsWith("[INFO]"))
		   {
			   return logLine.Remove(0,7).Trim();
		   }
		else if(logLine.StartsWith("[ERROR]"))
			{
				return logLine.Remove(0,8).Trim();
			}
		else if(logLine.StartsWith("[WARNING]"))
			{
				return logLine.Remove(0,10).Trim();
			}
		else
			{
				return ("program failed");
			}
    }
    public static string LogLevel(string logLine)
    {
        if(logLine.StartsWith("[INFO]"))
		   {
			   return "info";
		   }
		else if(logLine.StartsWith("[ERROR]"))
			{
				return "error";
			}
		else if(logLine.StartsWith("[WARNING]"))
			{
				return "warning";
			}
		else
			{
				return ("program failed");
			}
    }
    public static string Reformat(string logLine)
    {
        if(logLine.StartsWith("[INFO]"))
		   {
		   return (logLine.Remove(0,7).Trim()+" (info)");
		   }
		else if(logLine.StartsWith("[ERROR]"))
			{
				return (logLine.Remove(0,8).Trim()+" (error)");
			}
		else if(logLine.StartsWith("[WARNING]"))
			{
				return (logLine.Remove(0,10).Trim()+" (warning)");
			}
		else
			{
				return ("program failed");
			}
    }
}
